README.md

```diff
@@ -1,4 +1,5 @@
-Cuberite [![Build Status](https://img.shields.io/travis/cuberite/cuberite/master.svg?style=flat)](https://travis-ci.org/cuberite/cuberite) [![Coverity Scan Build Status](https://img.shields.io/coverity/scan/1930.svg)](https://scan.coverity.com/projects/1930)
+Cuberite [![Build Status](https://img.shields.io/travis/cuberite/cuberite/master.svg?style=flat)](https://travis-ci.org/cuberite/cuberite) [![Circle Ci Build Status](https://circleci.com/gh/cuberite/cuberite.svg?&style=shield
+)](https://circleci.com/gh/cuberite/cuberite) [![AppVeyor Bild Status](https://ci.appveyor.com/api/projects/status/github/cuberite/cuberite?svg=true)](https://ci.appveyor.com/project/madmaxoft/mcserver)
```

Comment:
Hah, spotted a typo. 

In the first line, maybe put in Circle CI instead, but more importantly, it's AppVeyor *Build* instead of *Bild* 😛